base/precompile.jl

```diff
@@ -122,6 +122,7 @@ precompile(Tuple{typeof(Base.Filesystem.isdir), String})
 precompile(Tuple{typeof(Base.Filesystem.pwd)})
 precompile(Tuple{typeof(Base.Filesystem.splitdir), String})
 precompile(Tuple{typeof(Base.Filesystem.splitext), String})
+precompile(Tuple{typeof(Base.Filesystem.splitpath), String})
```

Comment:
I think this file is generated automatically so you can probably skip this line.

Reply:
Oh cool, okay thanks. Removed.